arduino/libraries/librariesmanager/install.go

```diff
@@ -85,3 +91,99 @@ func (lm *LibrariesManager) Uninstall(lib *libraries.Library) error {
 	lm.Libraries[lib.Name].Remove(lib)
 	return nil
 }
+
+//InstallZipLib  installs a Zip library on the specified path.
+func (lm *LibrariesManager) InstallZipLib(libPath string) error {
+	libsDir := lm.getUserLibrariesDir()
+	if libsDir == nil {
+		return fmt.Errorf("User directory not set")
+	}
+	err := Unzip(libPath, libsDir.String())
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+//Unzip takes the ZipLibPath and Extracts it to LibsDir
+func Unzip(src string, dest string) error {
```

Comment:
There's no need for this function to be public, it's used only inside `InstallZipLib`.
Also you can avoid repeating the type of the arguments, `src, dest string` is the preferred way.

Reply:
Yeah sure, I'll do the same with Fetch()?
It also is called only once in InstallGitLib.